src/app/recipe-list/recipe-list.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit } from '@angular/core';
+import { Observable, of } from 'rxjs';
+import { RecipeService } from '../services/recipe.service';
```

Comment:
Make sure to change this to RecipeService import to the correct location.

Reply:
I've already noticed that in my local workspace.